lib/hardware/modules/reg/iob_reg/iob_reg.py

```diff
@@ -23,7 +22,7 @@ def setup(py_params_dict):
             },
             {
                 "name": "RST_POL",
-                "type": "M",
+                "type": "F",
```

Comment:
why change to F ?

Reply:
this can be a python variable ; RST_POL applies to the whole design

Reply:
I also noticed this, but I wasn't sure how we were going to implement it globally in python.
Maybe using `sys.argv`?

Reply:
as top level macro ?

Reply:
i2s or png should have this